src/main/kotlin/net/atos/zac/signalering/SignaleringenJob.kt

```diff
@@ -0,0 +1,377 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.signalering
+
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.inject.Inject
+import jakarta.transaction.Transactional
+import net.atos.client.zgw.shared.util.URIUtil
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.client.zgw.ztc.ZTCClientService
+import net.atos.client.zgw.ztc.model.generated.ZaakType
+import net.atos.zac.configuratie.ConfiguratieService
+import net.atos.zac.flowable.TakenService
+import net.atos.zac.gebruikersvoorkeuren.model.TabelInstellingen
+import net.atos.zac.signalering.model.Signalering
+import net.atos.zac.signalering.model.SignaleringDetail
+import net.atos.zac.signalering.model.SignaleringTarget
+import net.atos.zac.signalering.model.SignaleringType
+import net.atos.zac.signalering.model.SignaleringVerzendInfo
+import net.atos.zac.signalering.model.SignaleringVerzondenZoekParameters
+import net.atos.zac.util.UriUtil
+import net.atos.zac.zaaksturing.ZaakafhandelParameterService
+import net.atos.zac.zoeken.ZoekenService
+import net.atos.zac.zoeken.model.DatumRange
+import net.atos.zac.zoeken.model.DatumVeld
+import net.atos.zac.zoeken.model.FilterVeld
+import net.atos.zac.zoeken.model.FilterWaarde
+import net.atos.zac.zoeken.model.ZoekObject
+import net.atos.zac.zoeken.model.ZoekParameters
+import net.atos.zac.zoeken.model.index.ZoekObjectType
+import net.atos.zac.zoeken.model.zoekobject.ZaakZoekObject
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import org.flowable.task.api.Task
+import java.time.LocalDate
+import java.util.Objects
+import java.util.UUID
+import java.util.function.Consumer
+import java.util.logging.Logger
+
+@ApplicationScoped
+@Transactional
+@Suppress("TooManyFunctions")
+@NoArgConstructor
+@AllOpen
+class SignaleringenJob @Inject constructor(
+    private val signaleringenService: SignaleringenService,
+    private val configuratieService: ConfiguratieService,
+    private val ztcClientService: ZTCClientService,
+    private val zaakafhandelParameterService: ZaakafhandelParameterService,
+    private val zoekenService: ZoekenService,
+    private val takenService: TakenService
+) {
+
+    companion object {
+        private val LOG: Logger = Logger.getLogger(SignaleringenJob::class.java.name)
```

Comment:
Trivial thing but you could remove the type here:
```suggestion
        private val LOG = Logger.getLogger(SignaleringenJob::class.java.name)
```

In future we could consider to switch to Kotlin Logging for Kotlin code but for now this is ok.